### PR TITLE
Customer checkbox final

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
@amuramoto final code for checkbox plugin tutorial, again, let's review this then I'll strip back to starter.

### CODE REVIEW

File: checkbox/app.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+ * To run this code, you must do the following:
+ *
+ * 1. Deploy this code to a server running Node.js
+ * 2. Run `npm install`
+ * 3. Update the VERIFY_TOKEN
+ * 4. Add your PAGE_ACCESS_TOKEN to your environment vars
+ *
+ */
+
+const PAGE_ACCESS_TOKEN = "";
```
Comment (amuramoto) [inline checkbox/app.js]: @ChrisChinchilla put var declarations below imports. Imports should be first thing below the license text

File: checkbox/app.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+// Imports dependencies and set up http server
+import request from 'request'; // creates express http server
+
```
Comment (amuramoto) [inline checkbox/app.js]: @ChrisChinchilla remove line break

File: checkbox/app.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+import express from 'express';
+import body_parser from 'body-parser';
+import path from 'path';
```
Comment (amuramoto) [inline checkbox/app.js]: @ChrisChinchilla Is this dependency needed? I'm not too familiar but it looks like your usage below of:

path.join(__dirname, '/public', 'thanks.html')

is the same as __dirname+'/public'+'/thanks.html';

If possible, let's avoid including any unnecessary dependencies.

File: checkbox/app.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+    } else {
+        // Return a '404 Not Found' if event is not from a page subscription
+        res.sendStatus(404);
+    }
+
+});
+
+// Accepts GET requests at the /webhook endpoint
+app.get('/webhook', (req, res) => {
```
Comment (amuramoto) [inline checkbox/app.js]: @ChrisChinchilla Let's move the verify endpoint to the bottom of the file since it's boilerplate.

**Comment by rubycalling** [comment]
Pull Request is too old (2018) - This will need to be re-assess.

**Pull request closed without merging**